skins/cat17/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "standard",
+  "plugins": [
+    "html"
+  ],
+  "env": {
+    "jasmine": true
```

Comment:
Until we use Jasmine as our testing framework, please remove this global.
We also might need to add `WMDE` here.